app/src/main/java/com/isc/hermes/utils/PlacesSearchedOnTheMap.java

```diff
@@ -0,0 +1,55 @@
+package com.isc.hermes.utils;
+
+import android.content.Context;
+import com.mapbox.mapboxsdk.annotations.MarkerOptions;
+import com.mapbox.mapboxsdk.geometry.LatLng;
+import com.mapbox.mapboxsdk.maps.MapboxMap;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This Class creates waypoints which will be marked on the map.
+ */
+public class PlacesSearchedOnTheMap {
+    private final MapboxMap mapboxMap;
+    List<LatLng> pointList = new ArrayList<>(); //TODO: Once the searcher functionality is added delete this line
+
+    /**
+     * This method is the class constructor.
+     *
+     * @param mapboxMap the MapboxMap object to be configured.
+     * @param context Get global information about the environment of an application.
+     */
+    public PlacesSearchedOnTheMap(MapboxMap mapboxMap, Context context) {
+        this.mapboxMap = mapboxMap;
+        //pointTest(); //TODO: uncomment this method to test the functionality then this line must be deleted
+        displayPoint(pointList);
+    }
+
+    /**
+     * This method creates and displays waypoints on the map.
+     * @param pointList list of places that will be marked on the map.
+     */
+    public void displayPoint(List<LatLng> pointList) {
+        for (int i = 0; i < pointList.size(); i++) {
```

Comment:
Use a **For each** to make it more efficient and clear.

Reply:
Se intento usar un for each, lastimosamente no funcionaba el programa